src/openforms/prefill/contrib/eidas/tests/test_config_checks.py

```diff
@@ -0,0 +1,114 @@
+            oidc_op_token_endpoint="http://localhost/oidc/token",
+            oidc_op_user_endpoint="http://localhost/oidc/userinfo",
+            oidc_op_logout_endpoint="http://localhost/oidc/logout",
+        )
+
+    def test_check_config_ok(self):
+        OFOIDCClientFactory.create(
+            enabled=True,
+            with_eidas=True,
+            oidc_provider=self.provider,
+        )
+        plugin = EIDASPrefill(identifier="eidas")
+
+        try:
+            plugin.check_config()
+        except InvalidPluginConfiguration as exc:
+            raise self.failureException("Config check should have passed") from exc
+
+    def test_no_eidas_OIDC_client_configured(self):
+        plugin = EIDASPrefill(identifier="eidas")
+
+        with self.assertRaises(InvalidPluginConfiguration) as exc:
+            plugin.check_config()
+
+        self.assertEqual(exc.exception.args[0], _("Missing OIDC client for eIDAS."))
+
+    def test_eidas_OIDC_client_disabled(self):
+        OFOIDCClientFactory.create(
+            enabled=False,
+            with_eidas=True,
+            oidc_provider=self.provider,
+        )
+        plugin = EIDASPrefill(identifier="eidas")
+
+        with self.assertRaises(InvalidPluginConfiguration) as exc:
+            plugin.check_config()
+
+        self.assertEqual(exc.exception.args[0], _("Missing OIDC client for eIDAS."))
+
+    def test_eidas_OIDC_client_missing_identity_settings(self):
+        OFOIDCClientFactory.create(
+            enabled=True,
+            with_eidas=True,
+            oidc_provider=self.provider,
+            options__identity_settings=None,
+        )
+        plugin = EIDASPrefill(identifier="eidas")
+
+        # Check the config and expect a raised exception
+        with self.assertRaises(InvalidPluginConfiguration) as exc:
+            plugin.check_config()
+
+        self.assertEqual(
+            exc.exception.args[0],
+            _("Missing OIDC client identity settings for eIDAS."),
+        )
+
+    def test_eidas_OIDC_client_missing_identity_settings_options(self):
+        client = OFOIDCClientFactory.create(
+            enabled=True,
+            with_eidas=True,
+            oidc_provider=self.provider,
+        )
+        plugin = EIDASPrefill(identifier="eidas")
+
+        required_fields = (
+            "legal_subject_identifier_claim_path",
+            "legal_subject_identifier_type_claim_path",
+            "legal_subject_first_name_claim_path",
+            "legal_subject_family_name_claim_path",
+            "legal_subject_date_of_birth_claim_path",
+        )
+
+        # Sanity check; the initial config is correct
+        try:
+            plugin.check_config()
+        except InvalidPluginConfiguration as exc:
+            raise self.failureException("Config check should have passed") from exc
+
+        # Each required_field should trigger an exception, when left empty
+        for field in required_fields:
```

Comment:
please use `with self.subTest(...)` when performing tests in a loop, it makes it easier to track down the failing case and ensures all cases are tried even though one may have failed